server/status.go

```diff
@@ -87,23 +86,16 @@ func (s *statusServer) registerHandlers(mux *http.ServeMux) {
 	mux.HandleFunc(statusTransactionsKeyPrefix, s.handleTransactionStatus)
 }
 
-// marshalJSON marshals the provided obj into indented JSON format.
-func (s *statusServer) marshalJSON(r *http.Request, obj interface{}) ([]byte, error) {
-	return json.MarshalIndent(obj, "", "  ")
-}
-
 // handleStatus handles GET requests for cluster status.
 func (s *statusServer) handleStatus(w http.ResponseWriter, r *http.Request) {
-	w.Header().Set("Content-Type", "application/json")
-
 	cluster := &status.Cluster{}
-
-	b, err := s.marshalJSON(r, cluster)
+	b, contentType, err := util.MarshalResponse(r, cluster, util.AllEncodings)
```

Comment:
Added `[]util.EncodingType{util.JSONEncoding}`. 

I initially tried the fallback approach, but I thought it might accidentally hide a bug (e.g., `MarshalResponse` falls back to JSON when a given YAML is bogus).